database-plugins/widgets/Database-batchsource.json

```diff
@@ -75,22 +83,6 @@
           "widget-type": "hidden",
           "name": "query"
         },
-        {
-          "widget-type": "plugin-list",
-          "label": "JDBC Driver name",
-          "name": "jdbcPluginName",
-          "widget-attributes": {
-            "plugin-type": "jdbc"
-          }
-        },
-        {
-          "widget-type": "textbox",
```

Comment:
removing this means the property shows up in the UI without any formatting. If we no longer want to show this (which makes sense to me), need to set the widget type to 'hidden'.

Reply:
We don't have this property any more at backend. It's now hardcoded to jdbc.

Reply:
removing a property is a backwards incompatible change. We don't want this plugin type in the new connector, but the source still needs to support it for compatibility. It's possible, though unlikely, that somebody uploaded a jdbc driver as some other plugin type (they can set it to whatever they want using the API). 

If it was removed earlier, please add it back, just for the source. Also set the widget-type to 'hidden' so that new pipelines never set it to a value.

Reply:
added back.